arlo-client/src/components/Audit/index.tsx

```diff
@@ -100,7 +112,9 @@ const Audit: React.FC<IProps> = ({
         updateAudit={updateAudit}
       />
 
-      {isAuthenticated ? (
+      {isAuthenticated &&
+      /* istanbul ignore next */
+      (stageMatch === 'setup' || stageMatch === 'progress') ? (
```

Comment:
I suggest constants for those strings. (Enum?)

Reply:
@benadida Just added a type constraint for this value in the `TParams` interface. Does that work?

Reply:
just to be clear, if you typo `setup` in the code here and say `setu`, will typescript complain? If so, I'm satisfied :)

Reply:
![image](https://user-images.githubusercontent.com/1938/76978842-7310a100-68f4-11ea-912f-c1b7ba65fffc.png)